ui/appui-abstract/src/appui-abstract/properties/Description.ts

```diff
@@ -86,8 +86,13 @@ export interface PropertyDescription {
   /** Quantity type key used to look up formatting and parsing specs. This is typically either the name of a quantity type used by a tool
    *  or the full name of a KOQ (schema:koq).
    * @alpha
+   * @deprecated in 5.0. Use `kindOfQuantityName` instead.
    */
   quantityType?: string;
```

Comment:
Should we deprecate `quantityType` getters of `BaseQuantityDescription` and subclasses: `FormattedQuantityDescription`, `AngleDescription`, `EngineeringLengthDescription`, `LengthDescription`, `SurveyLengthDescription`? Or all of these classes will be deprecated later on instead?

Reply:
Later, we'll have to start by deprecating `QuantityType` in core-frontend, and we've [documented](https://github.com/iTwin/itwinjs-core/blob/master/docs/learning/frontend/QuantityFormatting.md#migrating-from-quantitytype-to-kindofquantity) that plan to be done during 5.x lifecycle. We'll promote the new way of retrieving formats through the upcoming first party application, to encourage other products to do the same